distribution/docs/src/main/jdocs/content/_configuring/managing-certificates-contents.adoc

```diff
@@ -83,28 +83,47 @@ To use the scripts, run them out of the `<INSTALL_HOME>/etc/certs` directory.
 
 For *NIX, use the `CertNew.sh` script.
 
-`sh CertNew.sh <FQDN>`
+`sh CertNew.sh (-cn <cn>|-dn <dn>) [-san <tag:name,tag:name,...>]`
```

Comment:
Good catch. I copied and pasted everywhere when only the CertsNew.sh supports -cn and -dn to be optional. I corrected it everywhere including the doc.